project/api/management/commands/import_narratives.py

```diff
@@ -16,52 +16,58 @@ def dms2dd(degrees, minutes, seconds, direction):
             dd *= -1
         return dd
 
+    def parseQID(id_string):
+        id_list = id_string.split(",")
```

Comment:
I would propose to use a regexp and remove everything except commas and digits [0-9,] before using split, then there would be no need to 'parse'

Reply:
How is it now?

Reply:
I did not see any changes to this part